tests/units/reporter/test_csv.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+"""Test anta.report.csv_reporter.py."""
+
+# pylint: disable=too-few-public-methods
+
+import csv
+import pathlib
+from typing import Any, Callable
+
+from anta.reporter.csv_reporter import ReportCsv
+from anta.result_manager import ResultManager
+
+# To avoid such error:
+# pytest.PytestCollectionWarning: cannot collect test class 'TestResult' because it has a __init__ constructor
```

Comment:
I am not sure I understand the meaning of this comment - what is the problem here? I suppose it is because we have a class called TestResult in out code and pytest is doing some collection on it?

Reply:
Comment that can be removed since there is no more `TestResult` import